packages/api/src/command/medical/document/document-conversion-status.ts

```diff
@@ -70,7 +70,7 @@ export async function calculateDocumentConversionStatus({
     });
 
     if (isConversionCompleted) {
-      const startedAt = updatedPatient.data.documentQueryProgress?.startedAt;
+      const startedAt = externalData?.documentQueryProgress?.startedAt;
```

Comment:
Forgot about this but its affected by scheduling the DQ so need to get the HIE startAt